turbopy/diagnostics.py

```diff
@@ -184,19 +205,44 @@ def finalize(self):
 
 
 class ClockDiagnostic(Diagnostic):
+    """Diagnostic subclass used to store and save time data into a CSV file
+    using the CSVOutputUtility class.
+
+    Parameters
+    ----------
+    owner : Simulation
+        The 'Simulation' object that contains this object
+    input_data : dict
+        Dictionary containing information abut the class such as its name
+
+    Attributes
+    ----------
+    owner : Simulation
+        The 'Simulation' object that contains this object
+    input_data : dict
+        Dictionary containing information abut the class such as its name
+    filename : str
+        File name for CSV time file
+    csv : :class:'numpy.ndarray'
+        Array to store values to be written into a CSV file
+    """
+
     def __init__(self, owner: Simulation, input_data: dict):
         super().__init__(owner, input_data)
         self.filename = input_data["filename"]
         self.csv = None
 
     def diagnose(self):
+        """Append time into the self.csv ndarray."""
         self.csv.append(self.owner.clock.time)
 
     def initialize(self):
+        """Initialize self.csv as an instance of the CSVOuputUtility class."""
```

Comment:
I'm not sure if this is the correct notation, but can you add the code that that `CSVOutputUtility` automatically gets linked?

```suggestion
        """Initialize `self.csv` as an instance of the :class:`CSVOuputUtility` class."""
```